testSoftMxRemote configuration failure JDK8

The JDK8 extended tests have been failing since Jan 17. I suspect its caused by #793 un-excluding the test. @smlambert fyi

https://ci.eclipse.org/openj9/job/Test-Extended-JDK8-linux_390-64_cmprssptrs/
https://ci.eclipse.org/openj9/job/Test-Extended-JDK8-linux_ppc-64_cmprssptrs_le/

```
02:04:35 FAILED CONFIGURATION: @BeforeMethod setUp
02:04:35 java.lang.AssertionError: The server is not ready after 120 seconds!!!
02:04:35 	at org.testng.Assert.fail(Assert.java:94)
02:04:35 	at j9vm.test.softmx.SoftmxRemoteTest.waitRemoteServer(SoftmxRemoteTest.java:344)
02:04:35 	at j9vm.test.softmx.SoftmxRemoteTest.setUp(SoftmxRemoteTest.java:71)
02:04:35 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
02:04:35 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
02:04:35 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
02:04:35 	at java.lang.reflect.Method.invoke(Method.java:498)
02:04:35 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
02:04:35 	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
02:04:35 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
02:04:35 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
02:04:35 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
02:04:35 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
02:04:35 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
02:04:35 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
02:04:35 	at org.testng.TestRunner.privateRun(TestRunner.java:756)
02:04:35 	at org.testng.TestRunner.run(TestRunner.java:610)
02:04:35 	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
02:04:35 	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
02:04:35 	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
02:04:35 	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
02:04:35 	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
02:04:35 	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
02:04:35 	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
02:04:35 	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
02:04:35 	at org.testng.TestNG.runSuites(TestNG.java:1133)
02:04:35 	at org.testng.TestNG.run(TestNG.java:1104)
02:04:35 	at org.testng.TestNG.privateMain(TestNG.java:1434)
02:04:35 	at org.testng.TestNG.main(TestNG.java:1403)
02:04:35 
02:04:35 SKIPPED CONFIGURATION: @AfterMethod tearDown
02:04:35 SKIPPED CONFIGURATION: @BeforeMethod setUp
02:04:35 SKIPPED CONFIGURATION: @AfterMethod tearDown
02:04:35 SKIPPED CONFIGURATION: @BeforeMethod setUp
02:04:35 SKIPPED CONFIGURATION: @AfterMethod tearDown
02:04:35 SKIPPED CONFIGURATION: @BeforeMethod setUp
02:04:35 SKIPPED CONFIGURATION: @AfterMethod tearDown
02:04:35 SKIPPED: testJMXSetHeapSize
02:04:35 java.lang.AssertionError: The server is not ready after 120 seconds!!!
02:04:35 	at org.testng.Assert.fail(Assert.java:94)
02:04:35 	at j9vm.test.softmx.SoftmxRemoteTest.waitRemoteServer(SoftmxRemoteTest.java:344)
02:04:35 	at j9vm.test.softmx.SoftmxRemoteTest.setUp(SoftmxRemoteTest.java:71)
02:04:35 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
02:04:35 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
02:04:35 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
02:04:35 	at java.lang.reflect.Method.invoke(Method.java:498)
02:04:35 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
02:04:35 	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
02:04:35 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
02:04:35 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
02:04:35 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
02:04:35 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
02:04:35 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
02:04:35 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
02:04:35 	at org.testng.TestRunner.privateRun(TestRunner.java:756)
02:04:35 	at org.testng.TestRunner.run(TestRunner.java:610)
02:04:35 	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
02:04:35 	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
02:04:35 	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
02:04:35 	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
02:04:35 	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
02:04:35 	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
02:04:35 	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
02:04:35 	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
02:04:35 	at org.testng.TestNG.runSuites(TestNG.java:1133)
02:04:35 	at org.testng.TestNG.run(TestNG.java:1104)
02:04:35 	at org.testng.TestNG.privateMain(TestNG.java:1434)
02:04:35 	at org.testng.TestNG.main(TestNG.java:1403)
02:04:35 
02:04:35 SKIPPED: testJMXSetHeapSizeBiggerThanLimit
02:04:35 java.lang.AssertionError: The server is not ready after 120 seconds!!!
02:04:35 	at org.testng.Assert.fail(Assert.java:94)
02:04:35 	at j9vm.test.softmx.SoftmxRemoteTest.waitRemoteServer(SoftmxRemoteTest.java:344)
02:04:35 	at j9vm.test.softmx.SoftmxRemoteTest.setUp(SoftmxRemoteTest.java:71)
02:04:35 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
02:04:35 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
02:04:35 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
02:04:35 	at java.lang.reflect.Method.invoke(Method.java:498)
02:04:35 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
02:04:35 	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
02:04:35 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
02:04:35 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
02:04:35 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
02:04:35 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
02:04:35 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
02:04:35 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
02:04:35 	at org.testng.TestRunner.privateRun(TestRunner.java:756)
02:04:35 	at org.testng.TestRunner.run(TestRunner.java:610)
02:04:35 	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
02:04:35 	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
02:04:35 	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
02:04:35 	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
02:04:35 	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
02:04:35 	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
02:04:35 	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
02:04:35 	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
02:04:35 	at org.testng.TestNG.runSuites(TestNG.java:1133)
02:04:35 	at org.testng.TestNG.run(TestNG.java:1104)
02:04:35 	at org.testng.TestNG.privateMain(TestNG.java:1434)
02:04:35 	at org.testng.TestNG.main(TestNG.java:1403)
02:04:35 
02:04:35 SKIPPED: testJMXSetHeapSizeSmallerThanMin
02:04:35 java.lang.AssertionError: The server is not ready after 120 seconds!!!
02:04:35 	at org.testng.Assert.fail(Assert.java:94)
02:04:35 	at j9vm.test.softmx.SoftmxRemoteTest.waitRemoteServer(SoftmxRemoteTest.java:344)
02:04:35 	at j9vm.test.softmx.SoftmxRemoteTest.setUp(SoftmxRemoteTest.java:71)
02:04:35 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
02:04:35 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
02:04:35 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
02:04:35 	at java.lang.reflect.Method.invoke(Method.java:498)
02:04:35 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
02:04:35 	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
02:04:35 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
02:04:35 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
02:04:35 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
02:04:35 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
02:04:35 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
02:04:35 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
02:04:35 	at org.testng.TestRunner.privateRun(TestRunner.java:756)
02:04:35 	at org.testng.TestRunner.run(TestRunner.java:610)
02:04:35 	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
02:04:35 	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
02:04:35 	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
02:04:35 	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
02:04:35 	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
02:04:35 	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
02:04:35 	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
02:04:35 	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
02:04:35 	at org.testng.TestNG.runSuites(TestNG.java:1133)
02:04:35 	at org.testng.TestNG.run(TestNG.java:1104)
02:04:35 	at org.testng.TestNG.privateMain(TestNG.java:1434)
02:04:35 	at org.testng.TestNG.main(TestNG.java:1403)
02:04:35 
02:04:35 SKIPPED: testSoftmxHasEffects
02:04:35 java.lang.AssertionError: The server is not ready after 120 seconds!!!
02:04:35 	at org.testng.Assert.fail(Assert.java:94)
02:04:35 	at j9vm.test.softmx.SoftmxRemoteTest.waitRemoteServer(SoftmxRemoteTest.java:344)
02:04:35 	at j9vm.test.softmx.SoftmxRemoteTest.setUp(SoftmxRemoteTest.java:71)
02:04:35 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
02:04:35 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
02:04:35 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
02:04:35 	at java.lang.reflect.Method.invoke(Method.java:498)
02:04:35 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
02:04:35 	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
02:04:35 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
02:04:35 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
02:04:35 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
02:04:35 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
02:04:35 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
02:04:35 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
02:04:35 	at org.testng.TestRunner.privateRun(TestRunner.java:756)
02:04:35 	at org.testng.TestRunner.run(TestRunner.java:610)
02:04:35 	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
02:04:35 	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
02:04:35 	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
02:04:35 	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
02:04:35 	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
02:04:35 	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
02:04:35 	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
02:04:35 	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
02:04:35 	at org.testng.TestNG.runSuites(TestNG.java:1133)
02:04:35 	at org.testng.TestNG.run(TestNG.java:1104)
02:04:35 	at org.testng.TestNG.privateMain(TestNG.java:1434)
02:04:35 	at org.testng.TestNG.main(TestNG.java:1403)
02:04:35 
02:04:35 
02:04:35 ===============================================
02:04:35     testSoftMxRemote
02:04:35     Tests run: 4, Failures: 0, Skips: 4
02:04:35     Configuration Failures: 1, Skips: 7
02:04:35 ===============================================
02:04:35 
02:04:35 
02:04:35 ===============================================
02:04:35 JLM_Tests_Suite
02:04:35 Total tests run: 4, Failures: 0, Skips: 4
02:04:35 Configuration Failures: 1, Skips: 7
02:04:35 ===============================================
```

--- a/test/TestConfig/resources/excludes/latest_exclude_SE80.txt
+++ b/test/TestConfig/resources/excludes/latest_exclude_SE80.txt
@@ -30,3 +30,4 @@ org.openj9.test.attachAPI.TestAttachAPI:test_agntld02																	238 generi
 org.openj9.test.attachAPI.TestAttachAPI:test_agntld03																	238 generic-all
 org.openj9.test.attachAPI.TestSunAttachClasses:testAgentLoading															238 generic-all
 org.openj9.test.vmArguments.VmArgumentTests:testCrNocr																	244 generic-all
+j9vm.test.softmx.SoftmxRemoteTest																						480 generic-all

